.github/ISSUE_TEMPLATE.md

```diff
@@ -14,13 +14,15 @@ Hey there! Before you report a bug or suggest a new feature,
 > This is not the place to learn Java. Please refer to [StackOverflow][stack overflow]
   for your general programming questions.
 
-## Template
+## Description
```

Comment:
Maybe use "Issue" instead to avoid using "Description" twice?

Reply:
Maybe thats better yeah